docs/intro.md

```diff
@@ -0,0 +1,158 @@
+# QuickStart
+## Prerequisites
+Before installing nfs-provisioner make sure your Kubernetes cluster meets the following prerequisites:
+
+1. Kubernetes version 1.17
+2. NFS Client is installed on all nodes that will run a pod that mounts an `openebs-rwx` volume.
+   Here's how to prepare an NFS client on some common Operating Systems:
+
+| OPERATING SYSTEM |  How to install NFS Client package                                |
+| ---------------- | -------------------------------------------------------- |
+| RHEL/CentOS/Fedora  |run *sudo yum install nfs-utils -y*      |
+| Ubuntu/Debian   |run *sudo apt install nfs-common -y*     |
+| MacOS     |Should work out of the box |
+| FreeBSD  |Edit the */etc/rc.conf* file by setting or appending *nfs_client_enable="YES"* |
+| Windows   |1. Start PowerShell as Administrator.<br/>2. In case of Windows server, run *Install-WindowsFeature NFS-Client*<br/>3. In case of Windows host with a Desktop environment, run *Enable-WindowsOptionalFeature -FeatureName ServicesForNFS-ClientOnly, ClientForNFS-Infrastructure -Online -NoRestart* |
+
+## Install
+### Install NFS Provisioner through kubectl
+To install NFS Provisioner through kubectl, run below command:
+```
+kubectl apply -f deploy/kubectl/openebs-nfs-provisioner.yaml
```

Comment:
Can we add the yaml with the stable version here https://github.com/openebs/charts/tree/gh-pages like other engines and then the instruction can be

```
kubectl apply -f https://openebs.github.io/charts/openebs-nfs-provisioner.yaml
```

Reply:
I've raised a [PR](https://github.com/openebs/charts/pull/250)  to add YAML file  for v0.5.0

Reply:
Merged!

Reply:
Thanks!.
Updated the link to install the latest stable version.